src/library_wasmfs_jsimpl.js

```diff
@@ -84,7 +84,7 @@ mergeInto(LibraryManager.library, {
     assert(wasmFS$backends[backend]);
 #endif
     var result = await wasmFS$backends[backend].write(file, buffer, length, offset);
-    {{{ makeSetValue('result_p', 0, 'result', 'i64') }}};
+    {{{ makeSetValue('result_p', 0, 'result', 'i32') }}}; // TODO: wasm64
```

Comment:
```suggestion
    {{{ makeSetValue('result_p', 0, 'result', POINTER_TYPE) }}};
```
Thanks to @sbc100 's recent work it should be as simple as this I think.

Reply:
Nice, and I see there's `SIZE_TYPE` as well.